src/main/java/duke/command/Bye.java

```diff
@@ -0,0 +1,17 @@
+package duke.command;
+
+/**
+ * duke.command.Command to exit the program
```

Comment:
This seems to be an unintentional problem stemming from refactoring `Command` to `duke.command.Command`. Do you mind changing it?